src/NotionExporter.ts

```diff
@@ -40,18 +44,20 @@ export class NotionExporter {
     const id = validateUuid(blockIdFromUrl(idOrUrl))
     if (!id) return Promise.reject(`Invalid URL or blockId: ${idOrUrl}`)
 
+    const exportOptions = Object.assign({
+      exportType: "markdown",
+      timeZone: "Europe/Zurich",
+      locale: "en",
```

Comment:
Maybe we can allow in the future that these values are configurable? 🤔

Reply:
yep, I'll try to configure it in the next PR, event if it is _mixed_ with the package qualified name